cmd/root.go

```diff
@@ -75,8 +75,14 @@ func Execute() {
 	rootCmd.LocalFlags().Parse(os.Args)
 	b, _ := rootCmd.Flags().GetBool(argRootGenerate)
 	if b {
-		rootCmd.GenBashCompletionFile("./autocomplete/autocomplete.sh")
-		rootCmd.GenZshCompletionFile("./autocomplete/autocomplete.zsh")
+		err := rootCmd.GenBashCompletionFile("./autocomplete/autocomplete.sh")
```

Comment:
So the actual error was that I didn't have an `autocomplete` directory? We are now side-tracking the problem by adding the dir to git... :ok_hand:

Reply:
Yup, but, I guess, actually not ignoring the error doesn't hurt anyways